cross/ezstream/PLIST

```diff
@@ -0,0 +1,13 @@
+bin:bin/ezstream
+bin:bin/ezstream-file.sh
```

Comment:
shell scripts and xml files must be "rsc" for resources - not binary or shared object which will be stripped...